recipes/algotom/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "algotom" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/algotom/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256:  9a72ec5458c1604bbe4ae6e08f148cd27c3671b7df4a6cc9b4fdcda373176fca
+build:  
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
You should use `pip` here. See the recipe example or, better yet, re-generate this [recipe with grayskull](https://github.com/conda-incubator/grayskull).